docs/modules/ROOT/pages/lua/guide/InterComputerCommunication.adoc

```diff
@@ -148,20 +148,21 @@ net:open(42)
 
 while true do
  local data = {event.pull()}
- e, s, sender, port, data = function(e,s,sender,port,...)
+ e, s, sender, port, data = (function(e,s,sender,port,...)
   return e, s, sender, port, {...}
- end(table.unpack(data))
+ end) (table.unpack(data))
+ local d1, d2, d3, d4, d5, d6, d7 = table.unpack(data)
  if e == "NetworkMessage" then
   if d1 == "ping" then
```

Comment:
Rather than unpacking all possible data packets... use in the bottom if a `data[1]` instead... unpacking all of that is not what we want and rather useless.